layers/+lang/ruby/packages.el

```diff
@@ -149,6 +149,10 @@
     ;; setup the hook via an autoload
     :config
     (progn
+      (defun spacemacs//inf-ruby-auto-enter ()
+        "Automatically enters inf-ruby-mode in ruby modes' debugger breakpoints."
+        (add-hook 'compilation-filter-hook 'inf-ruby-auto-enter nil t))
+      (add-hook 'rspec-compilation-mode-hook 'spacemacs//inf-ruby-auto-enter)
```

Comment:
@syl20bnr It has to be made local to this hook, but other than that, your suggestion worked great. Thanks!